lib/ruby_lsp/requests/semantic_highlighting.rb

```diff
@@ -60,10 +60,13 @@ class SemanticHighlighting < BaseRequest
         default_library: 9,
       }.freeze, T::Hash[Symbol, Integer])
 
-      SPECIAL_RUBY_METHODS = T.let((Module.instance_methods(false) +
-        Kernel.methods(false) + Bundler::Dsl.instance_methods(false) +
```

Comment:
`Kernel.methods(false)` was already on the list, did you mean `Kernel.instance_methods(false)` in the PR description?

Reply:
Yes, that was a mistake on my part. I meant `Kernel.instance_methods(false)`.